LifeLine_App/.env.example

```
@@ -0,0 +1,11 @@
+# MAPS API KEY
+GOOGLE_MAPS_API_KEY=
+
+# GEOCODING API KEY 
+EXPO_PUBLIC_GEOCODING_API_KEY=
+
+# Google WEB Client ID
+EXPO_PUBLIC_WEB_CLIENT_ID=6
+
+# Development
+EXPO_PUBLIC_API_BASE_URL=
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)